database/database.go

```diff
@@ -22,5 +22,11 @@ func Migrate(databaseURL string) error {
 	if err != nil {
 		return err
 	}
-	return m.Up()
+
+	if err := m.Up(); err != nil && err != migrate.ErrNoChange {
+		return err
+	}
+
+	_, err = m.Close()
```

Comment:
Shouldn't this be in a `defer`?

Reply:
it doesn't really matter in this case as failed migration would mean the program will have to exit. There is no way it will be able to do anything with incorrect migration.
so it's just simpler to write like this than making named return with defer.

if you really want to change just to make it "completely right" let me know, I'll update the pr.

Reply:
no, it's fine 👍